e2e_playwright/st_popover.py

```diff
@@ -24,12 +24,12 @@
 
 st.popover("popover 1 (empty)")
 
-with st.popover("popover 2 (use_container_width)", use_container_width=True):
+with st.popover("popover 2 (use_container_width)", width="stretch"):
     st.markdown("Hello")
 
 with st.popover(
     "popover 9 (use_container_width) with help",
-    use_container_width=True,
+    width="stretch",
     help="help text",
```

Comment:
nit: the labels here with `(use_container_width)` aren't fully correct anymore. But maybe we can even remove at least the popover 2 case since its also covered in the new case